### ISSUE TITLE
support for compound data types

### ISSUE THREAD

**Issue by js216** [js216/CeNTREX#90]

**Comment by ograsdijk** [comment]
Added support in c1340702bd3cb21cfcae708e35e63345ca76fd87.
Add compound_dataset = True to the ini for devices which require compound datatypes. The driver should subsequently have self.dtype = ('f4', 'f8', ...) for instance, depending on the amount of returned values from ReadValue().
Only supported for slow_data so far, although I don't see any implementation for fast data so far.

Upon implementing I also noticed the HDF writer module might have issues with a queue with multiple entries, but I put all changes under if dev.config["compound_dataset"]: statements to prevent any breaking of legacy code in case I'm misunderstanding how the multiple entry data is handled.
